<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/blanco_delgado/Iconos/logo.png</normaloff>:/blanco_delgado/Iconos/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

QToolButton {
color: white;
background: #85C1E9;
border: none;
border-radius: 40px;
}

QToolButton:hover {
background:#3498DB;
}

QToolButton:pressed {
background: #DB7734;
}

#btnCerrarSesion {
border-radius: 25px;
}

QLabel {
color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="btnCerrarSesion">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Cerrar sesi√≥n</string>
         </property>
         <property name="icon">
          <iconset resource="recursos.qrc">
           <normaloff>:/blanco_delgado/Iconos/blanco_delgado/arrow-left-circle.svg</normaloff>:/blanco_delgado/Iconos/blanco_delgado/arrow-left-circle.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>722</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>202</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblMensaje">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>201</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <property name="spacing">
             <number>20</number>
            </property>
            <item row="0" column="0">
             <widget class="QToolButton" name="btnCrearEntidades">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Crear entidades</string>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/blanco_delgado/Iconos/blanco_delgado/database.svg</normaloff>:/blanco_delgado/Iconos/blanco_delgado/database.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
              <property name="autoRaise">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="btnAgregarRegistros">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Agregar registros</string>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/blanco_delgado/Iconos/blanco_delgado/file-plus.svg</normaloff>:/blanco_delgado/Iconos/blanco_delgado/file-plus.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="btnLeerRegistros">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Leer registros</string>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/blanco_delgado/Iconos/blanco_delgado/book-open.svg</normaloff>:/blanco_delgado/Iconos/blanco_delgado/book-open.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QToolButton" name="btnModificarRegistros">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Modificar registros</string>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/blanco_delgado/Iconos/blanco_delgado/edit.svg</normaloff>:/blanco_delgado/Iconos/blanco_delgado/edit.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="btnEliminarRegistros">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Eliminar registros</string>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/blanco_delgado/Iconos/blanco_delgado/file-minus.svg</normaloff>:/blanco_delgado/Iconos/blanco_delgado/file-minus.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QToolButton" name="btnGenerarConsultas">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Generar consultas</string>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/blanco_delgado/Iconos/blanco_delgado/book.svg</normaloff>:/blanco_delgado/Iconos/blanco_delgado/book.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
